templates/_worker.js/index.ts

```diff
@@ -54,7 +54,10 @@ export const routesMatcher = (
 		}
 
 		if ('src' in route) {
-			const regExp = new RegExp(route.src, caseSensitive ? undefined : 'i');
+			const regExp = new RegExp(
+				route.src as string,
```

Comment:
```suggestion
				route.src!,
```
Not too important since this stuff is going to go await with the routing change but `as string` makes it look like `route.src` could be something other than a string, I think the correct thing here would be to use a non null assertion instead

although it makes eslint complain... generally I'm not sure about `@typescript-eslint/no-non-null-assertion`, what do you think? should we disable it? (personally I don't see anything wrong with non-null assertions when used correctly)